cmake/modules/FindLIQ.cmake

```diff
@@ -4,15 +4,16 @@
 # This module defines
 #  LIQ_INCLUDE_DIR, where to find libimagequant.h
 #  LIQ_LIBRARIES, the libraries to link against to use libimagequant.
-#  LIQ_FOUND, If false, do not try to use libimagequant.
+#  LIQ_FOUND, if false, do not try to use libimagequant.
+#  LIQ_BUILD, if undefined, libgd will not download and build libimagequant
```

Comment:
did you mean "will download" ?

i'm kind of the opinion we should delete all this logic ... gd shouldn't be downloading random code & executing it from the internet on people

Reply:
> did you mean "will download" ?

If LIQ_BUILD is **undefined**, libgd will not download libimagequant. Correct?

> i'm kind of the opinion we should delete all this logic ... gd shouldn't be downloading random code & executing it from the internet on people

So we should not download libimaequant if it is not found, right?

Reply:
> If LIQ_BUILD is undefined, libgd will not download libimagequant. Correct?

sorry, you're right of course.  double negatives in programming often throw me off which is why i suggested people avoid them :).

> So we should not download libimaequant if it is not found, right?

imo, yeah, we should delete all of `LIQ_BUILD` and everything in the `ELSE (LIQ_FOUND)` section.

Reply:
Okey. I will improve it soon.

It would be greatly appreciated that you could spare some to review my rest PRs. I am close to finish building with cmake and ms toolchain under windows platform. And then we can close issue #249, which is an import part of milestone 2.2.6.